### ISSUE TITLE
Why do some users prefer FR1?

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/MyopicVicar#1070]
Problem with Website:

"How can i access version one as much easier to find siblings than version 2 which is cumbersome to use slow and very annoyingly slow from time clicking accept cookies to allowing one to type into any of the boxes and why does it give a message as one leaves a box re upper/lower case irrelevant, bring back version one and up date it not this slow cumbersome over prettified slow difficult to resolve any useful information from Please give me a link to version One it may be out of date but it did work fast and was easier to locate family groups on unlike version 2. Unhappy user"

**Comment by DeniseColbert** [comment]
When asked for clarification:
"Hi Denise
I have always used www.freereg.org.uk  and was sad when you changed to version 2, which is slow cumbersome and less helpful in use also if you use the browser back arrow the search block remains a square and you cannot search but have to forward and use the new search or edit keys why?, and yes I do prefer and would like to use version 1"

**Comment by Captainkirkdawson** [comment]
Interesting that they talk about the cookie! We are technically illegal with FR1 by not giving a cookie warning.
We could make the cookie process easier by going to implicit acceptance instead of  explicit acceptance as suggested in 992

**Comment by AlOneill** [comment]
We *could* move the Soundex, Family Members and Witnesses checkboxes to be near the Names inputs (first fieldset, with Dates), then put County, Place and Nearby places, together with Search button, in fieldset 2 ... or ... if that would be clearer, or more helpful ???

**Comment by Captainkirkdawson** [comment]
Is there some whitespace between the heading and the first box that could go?
So we need to search options box?

Why not 1 box?

**Comment by edickens** [comment]
I don't think FR1 uses Cookies.  It is very basic.  E
    
    On 30/01/2017 23:56, Kirk Dawson wrote:
    
    
      Interesting that they talk about the cookie! We are technically
        illegal with FR1 by not giving a cookie warning.
        We could make the cookie process easier by going to implicit
        acceptance instead of explicit acceptance as suggested in 992
      
      
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/FreeUKGen/MyopicVicar","title":"FreeUKGen/MyopicVicar","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/FreeUKGen/MyopicVicar"}},"updates":{"snippets":[{"icon":"PERSON","message":"@Captainkirkdawson in #1070: Interesting that they talk about the cookie! We are technically illegal with FR1 by not giving a cookie warning.\nWe could make the cookie process easier by going to implicit acceptance instead of  explicit acceptance as suggested in 992"}],"action":{"name":"View Issue","url":"https://github.com/FreeUKGen/MyopicVicar/issues/1070#issuecomment-276230889"}}}

**Comment by Captainkirkdawson** [comment]
No it does not.

**Comment by AlOneill** [comment]
White-space between heading and fieldset could be reduced by 14px, but this would give a slightly squashed appearance. (Would be surprised if that solved anything for the above complainer.)

Two fieldsets (boxes) are used to break form into manageable chunks. As noted above, we may not currently have the best field groupings and placement for ease of use and understanding.

Perhaps we need to work a bit harder on selling FR2. Change is harder to take when user has little idea of why it is needed and what the net benefits are. Article could cover such things as increased flexibility of new search approach, the need to accommodate a variety of devices, the need to be accessible to those with disabilities, the availability of up to date and accurate numbers of records, benefits to transcribers and coords (and hence for researchers), &c, &c.

**Comment by edickens** [comment]
From what I have heard, it is the presentation of the results and
    the text colour that are the two complaints.
    
    Take these search results..
    
    
    
    Click the 1 and you get:-
    
    
    
    All shown without scrolling.
    
    E
    
    On 31/01/2017 16:51, AlOneill wrote:
    
    
      White-space between heading and fieldset could be reduced by
        14px, but this would give a slightly squashed appearance. (Would
        be surprised if that solved anything for the above complainer.)
      Two fieldsets (boxes) are used to break form into manageable
        chunks. As noted above, we may not currently have the best field
        groupings and placement for ease of use and understanding.
      Perhaps we need to work a bit harder on selling FR2. Change is
        harder to take when user has little idea of why it is needed and
        what the net benefits are. Article could cover such things as
        increased flexibility of new search approach, the need to
        accommodate a variety of devices, the need to be accessible to
        those with disabilities, the availability of up to date and
        accurate numbers of records, benefits to transcribers and coords
        (and hence for researchers), &c, &c.

**Comment by Captainkirkdawson** [comment]
There are no images so  its tough to comment.

It is important to keep this in context. We are likely talking about less than 2% of the REG researchers.

At least 50% of the accesses to freereg1 leave without doing anything. They bounce. Likely are robots.

**Comment by edickens** [comment]
I agree with Kirk.  And remember that FR1 never had Places and Churches 
set up as a list, it just took whatever was transcribed including a load 
of rubbish.  So FR is far more comprehensive and will get even better.  
It is different and we cannot compare like with like.  FR1 was 
exceedingly complicated and error prone to maintain.

E

**Comment by AlOneill** [comment]
That -- "FR1 never had Places and Churches set up as a list, it just took whatever was transcribed including a load of rubbish." and "FR1 was exceedingly complicated and error prone to maintain." -- is just the kind of thing I had in mind to tell researchers. Though probably not in those words exactly!

2% of 14,000 say, is still 280 people per day who are far from convinced that FR2 is necessary or helpful. Maybe some will never be convinced, but others will get it.

**Comment by Captainkirkdawson** [comment]
The 14000 is searches per day not people.

Kirk Dawson
5220 Riverside Drive
Fairmont Hot Springs, B.C.
V0B 1L1

**Comment by AlOneill** [comment]
Oops! You are right, Kirk. Just trying to make the point that small percentages can hide larger numbers of things and/or people.

**Comment by PatReynolds** [comment]
As actions from the answer 'because of accessibility issues' are tabled elsewhere on the board, I am assigning this to 1.7, for consideration of the other actions at that time.

**Comment by PatReynolds** [comment]
Answer: (for a quarter) because they live in Edmonton:
![image.png](https://images.zenhubusercontent.com/5b695a5dac3a200155c765e3/40461618-ebc3-4412-adb1-cc6ca8811027)
(Analytics for last 30 days)

**Comment by Captainkirkdawson** [comment]
That `Answer: (for a quarter) because they live in Edmonton:` would be a funny to Canadians!!
A quarter is 25 cents and Edmontonians are always thought to be odd for living in one of the coldest cities in Canada.
This just proves it!

**Comment by stsccfr** [comment]
Closing as no longer relevant.
